index.html

```diff
@@ -3398,6 +3322,12 @@ <h3>The <dfn>EventSubscriptionHandler</dfn> callback</h3>
                 Create an <a>Event</a> notification |response| according to the
                 <a>Protocol Bindings</a> from |data| and |subscriber|, including
                 its |options|.
+              </li>
+              <li>
+                  If |data| is `undefined`, assume that the notification |response| 
+                  will contain a `void` data payload. 
```

Comment:
I suggest reformulating to more exact prose, to something like

> If |data| is `undefined`, let |response|'s |data| be `undefined`

(needs to be done consistently over all occurrences of |response|

Reply:
Call of 2022-08-08
@relu91 will work (provide) an alternative to make clear what `void` means..